docs/dice_verification.md

```diff
@@ -1,40 +1,137 @@
+<kbd><img src="img/dicedoc/coleman_addresses.png"></kbd>
+Check that the receive addresses match.<br>
+To verify the change addresses change 'External / Internal' to 1 (1):
+<kbd><img src="img/dicedoc/coleman_change_addresses_1.png"></kbd><br>
+Compare the change addresses to the ones generated in Sparrow (Addresses tab of the wallet):
+<kbd><img src="img/dicedoc/coleman_change_addresses_2.png"></kbd>
+Check that the change addresses match.
+
+<br><br>
+
+### Verifying with Seed Tool website
+
+Go to https://bitcoiner.guide/seed/ and click on 'Seed Generation Input' (1):<br>
+<kbd><img src="img/dicedoc/seedtool_1.png"></kbd>
+
+Then click on the 'Show the Entropy Section' tab (1):<br>
+<kbd><img src="img/dicedoc/seedtool_2.png"></kbd>
+
+Enter the 99 dice numbers in (1), in (2) change back to 'Hex', check that (3) is still '24 Words' and the calculated seed words are shown in (4):
+<kbd><img src="img/dicedoc/seedtool_3.png"></kbd>
+
+Seed words shown are the same as in Seedsigner and the Ian Coleman web tool seen before.
+<br>
+
+#### Verification of (1) fingerprint, (2) zpub and (3) generated addresses
+
+1. Fingerprint can be seen here (1): <br>
+<kbd><img src="img/dicedoc/seedtool_fingerprint.png"></kbd>
+1. Zpub:<br>
+Scroll down to the 'Derived Addresses' section (1), click on it, make sure that '84' is selected for 'Purpose' (2) and check the zpub at (3):<br>
+<kbd><img src="img/dicedoc/seedtool_zpub.png"></kbd><br>
+Compare to zpub in Sparrow:<br>
+<kbd><img src="img/dicedoc/sparrow_zpub.png"></kbd>
+Zpub is the same as shown in Seedsigner, Sparrow and Ian Colemand tool.
+1. Addresses:<br>
+Scroll down a little bit where the receive addresses are shown and compare to the ones generated in Sparrow:<br>
+<kbd><img src="img/dicedoc/seedtool_addresses.png"></kbd>
+Check that the receive addresses match.<br>
+To verify the change addresses change the 'Receive/Change' dropdown box to '1 (Change)' (1):
+<kbd><img src="img/dicedoc/seedtool_change_addresses_1.png"></kbd><br>
+Compare the change addresses to the ones generated in Sparrow (Addresses tab of the wallet):
+<kbd><img src="img/dicedoc/seedtool_change_addresses_2.png"></kbd>
+Check that the change addresses match.
+<br><br>
 
 ### Verifying with Coldcard
 
 There is nothing specific, the algorithm are completely the same. Coldcard has a verification script in python and all explanations here:
 https://coldcard.com/docs/verifying-dice-roll-math
+<br><br>
 
 ### Epilogue
-You can use these methods to do dry run time to time to verify that no one has changed the micro sdcard. But do not use the generated 24 words as a valid wallet, they need to be generated alone, only on the seedsigner!
+You can use these methods to do dry runs from time to time to verify that no one has changed the micro sdcard. But do not use the generated 24 words as a valid wallet, they need to be generated alone, only on the seedsigner!
```

Comment:
Perhaps use the word _conclusion_ here.  
Let us inform the reader, (who may be a little lost still) , that we have achieved a success if the Fingerprints, Addresses and zpubs all match up, and some celebration and congratulations are in order.
  :)